A/ApproxFunOrthogonalPolynomials/Compat.toml

```diff
@@ -45,3 +45,27 @@ SpecialFunctions = "0.7.0-*"
 StaticArrays = "0.8.3-*"
 ToeplitzMatrices = "0.4.0-*"
 julia = "0.7.0-*"
+
+["0.0.3"]
+AbstractFFTs = "0.3.1-0.4"
+ApproxFunBase = "0.0.4"
+BandedMatrices = "0.9"
+BlockArrays = "0.8"
+BlockBandedMatrices = "0.4"
+Calculus = "0.1.15-0.4"
+DSP = "0.5.1-0.5.2"
+DomainSets = "0.0"
+DualNumbers = "0.4-0.6"
+FFTW = "0.2.4"
+FastGaussQuadrature = "0.3.2-0.3.3"
+FastTransforms = "0.4.2"
+FillArrays = "0.5-0.6"
+InfiniteArrays = "0.0.3"
+IntervalSets = "0.3.1"
+LazyArrays = "0.8"
+LowRankApprox = "0.1.4-0.2"
+Reexport = "0.2"
+SpecialFunctions = "0.7"
+StaticArrays = "0.8.3-0.10"
+ToeplitzMatrices = "0.4-0.5"
+julia = "0.7-1.1"
```

Comment:
This is a bug https://github.com/JuliaComputing/Registrator.jl/issues/103 so please retrigger the bot when the linked issue is fixed.

Reply:
Is there something I should do in the meantime for this to be merged?

Reply:
Try retrigger, think it has been fixed.

Reply:
You can retrigger it now and it should be fixed.

Reply:
Hmm, the "fix" `julia = "0.7-1.3"` seems to also be wrong unless I'm missing something. I was expecting it to be compatible for all `1.x` versions.

Reply:
Yes, it is wrong.

Reply:
That's why I want to deploy the new version compression scheme because it fixes this.